controller/controller.js

```diff
@@ -1,12 +1,23 @@
-const { SelectCategories } = require("../__model/model");
+const { SelectCategories, SelectReviews } = require("../__model/model");
 
 exports.getCategories = (req, res, next) => {
-   SelectCategories().then((response) => {
-    res.status(200).send(response)
-   })
-   .catch((err) => {
+  SelectCategories()
+  .then((response) => {
+    res.status(200).send(response);
+  })
+  .catch((err) => {
     console.log(err, "controller reject");
     next(err);
-   
-});
+  });
+}
+
+exports.getReviews = (req, res, next) => {
+    SelectReviews()  
+    .then((response) => {
+        res.status(200).send(response);
```

Comment:
we **always** want to be sending our response back as an object with a descriptive key - i.e. here you'd want something like .send({reviews: response})